QuickViewer/src/mainwindow.ui

```diff
@@ -1015,7 +1015,7 @@
      <normaloff>:/icons/subfolders_icon</normaloff>:/icons/subfolders_icon</iconset>
    </property>
    <property name="text">
-    <string extracomment="When enumerating images, subfolders are also included in the range">Open with Subfolders Image</string>
+    <string extracomment="When enumerating images, subfolders are also included in the range">Open images from sub-folders as well</string>
```

Comment:
I don't understand what this one should be doing. 
The program is always includes every image of archive, even from sub-folders, isn't it?

Reply:
This feature works when Quick Viewer has an image in a folder open. As you say, it's always valid (can't be turned off) for images in the archive